server/src/internalClusterTest/java/org/elasticsearch/snapshots/RepositoriesIT.java

```diff
@@ -225,4 +229,106 @@ public void testRepositoryVerification() {
             assertThat(ExceptionsHelper.stackTrace(ex), containsString("is not shared"));
         }
     }
+
+    public void testSnapshotDeleteWithRepositoryOutage() throws Exception {
```

Comment:
I think this test is a little redundant, what does it cover that your second test doesn't?

Reply:
Agree. Removed the test